app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```

Comment:
> the same in other similar cases

Thanks. I've already changed. In one line I can't specify the arguments in any case, because the flake8 error occurs.